### PR TITLE
EIP-1191: extends EIP-55 by optionally adding a chain id defined by EIP-155 to the checksum calculation

### PR DESCRIPTION
**Pull request by juli**

### CODE REVIEW

File: EIPS/eip-1186.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+eip: 1186
```
Comment (Arachnid) [inline EIPS/eip-1186.md]: Why is this numbered 1186?

Comment (juli) [reply EIPS/eip-1186.md]: Tell me what number to use. 1191?

Comment (Arachnid) [reply EIPS/eip-1186.md]: Yes, 1191 is fine.

**Comment by axic** [comment]
@juli this EIP doesn't have a copyright statement. Can you please add one? EIP-1 suggests CC0 - see the `eip-x.md` template.

**Pull request merged**